### PR TITLE
Makes autoinjectors open again

### PR DESCRIPTION
**Pull request by robosperg**
repoban Horatio for giving me this idea

Alternatively, make vox/plasman suits give no resist to harm syringes
:cl:
 * rscadd: Autoinjectors are open containers again.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -87,9 +87,10 @@
 
 /obj/item/weapon/reagent_containers/hypospray/autoinjector/attack(mob/M as mob, mob/user as mob)
 	..()
-//	if(reagents.total_volume <= 0) //Prevents autoinjectors to be refilled.
-//		flags &= ~OPENCONTAINER
+	if(reagents.total_volume <= 0) //Prevents autoinjectors to be refilled.
```
Comment (boy2mantwicethefam) [inline code/modules/reagents/reagent_containers/hypospray.dm]: Remove comment

**Comment by boy2mantwicethefam** [comment]
BASED

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 1 above)
Comment (robosperg) [reply code/modules/reagents/reagent_containers/hypospray.dm]: ehhhhhh

**Comment by SonixApache** [comment]
literally why

**Comment by PunishedHoratio** [comment]
I'm Horatio and I approve of this change.

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -87,9 +87,10 @@
 
 /obj/item/weapon/reagent_containers/hypospray/autoinjector/attack(mob/M as mob, mob/user as mob)
 	..()
-//	if(reagents.total_volume <= 0) //Prevents autoinjectors to be refilled.
-//		flags &= ~OPENCONTAINER
+	if(reagents.total_volume <= 0) //Prevents autoinjectors to be refilled after use.
+		flags &= ~OPENCONTAINER
 	update_icon()
+	return
```
Comment (Exxion) [inline code/modules/reagents/reagent_containers/hypospray.dm]: Why

Comment (robosperg) [reply code/modules/reagents/reagent_containers/hypospray.dm]: I see code I copypaste code
https://github.com/vgstation-coders/vgstation13/pull/18725/files

**Comment by Kharkov72** [comment]
bro that's kinda cringe

**Comment by Blithering** [comment]
You feel a tiny prick!
You can almost hear someone talking...

**Comment by FudgePucker** [comment]
Cool, now you can have upgraded auto injectors for your medical staff, 3u dex+ 1u tramadol 1u bicarb

**Comment by FudgePucker** [comment]
As it stands autoinjecors have little use because 5u of DD isn't anything much

**Comment by ShiftyRail** [comment]
Harm syringes were buffed already, use that instead.

**Comment by aceedex** [comment]
The PR the server doesnt want you to know about

**Comment by SonixApache** [comment]
still waiting for the motivation in the OP

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 2 above)
Comment (Exxion) [reply code/modules/reagents/reagent_containers/hypospray.dm]: Well, remove this line

**Comment by iFlashYou** [comment]
>Harm syringes were buffed already, use that instead.

Laughs in plasmaman/vox/sec/engineer

**Comment by Dacendeth** [comment]
@iFlashYou most plasmaman/vox suits have little to no armor on them so they're still vulnerable to harmstabs.

**Comment by bathosbathtime** [comment]
Would it be possible to have autoinjectors blocked by the same things as harm syringes so that I can at least have useful quick injections of dex+ back.

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 2 above)
Comment (robosperg) [reply code/modules/reagents/reagent_containers/hypospray.dm]: done

**Comment by tuypo1** [comment]
Autoinjectors should have invaprop anyway

**Comment by Coxswain-Navigator** [comment]
This is a terrible idea and here's why. I personally abused the hell out of this feature when we had it in the past so i have a pretty good idea of how broken it is. It's basically infinite free paralysis pens is what it is. All that's requried is to make some neurotoxin with ghetto chem or steal some zombie powder from the medbay vends and you've got a parapen for every 5 units. As of right now, there aren't a lot of easy ways to get stuns which also sleep which means that this is the method EVERYONE will use. This will get boring quick and you'll wish you never merged this. It will be reverted if it ever passes.

**Comment by MadmanMartian** [comment]
WRONG ROBO

**Comment by SonixApache** [comment]
suggested alternative based on general discussion:

multiple injector types, easily switchable with some sort of medbay device

**Comment by tuypo1** [comment]
>multiple injector types, easily switchable with some sort of medbay device

sounds good

**Comment by Kurfursten** [comment]
Make it a medbay machine, we need more machinery.

**Comment by Gazathor** [comment]
The Best solution would be the ability to refill them with "reputable" drugs in a special machine, but thats just me screaming at the sun

**Comment by FudgePucker** [comment]
I dont know why this has such a backlash.
'Oh no, power gamers wearing hardsuits wont have total resistance from chemical injection attacks outside of a traitor specific item and the hypospray that there is only one of locked up in a locker in a heads office, this will RUIN EVERYTHING'

**Comment by Kurfursten** [comment]
Fudge, you can get autoinjectors in every first aid kit, including the ones that spawn in O2 lockers.

**Comment by FudgePucker** [comment]
Yes, and you also need something to remove the chemicals from it and get ahold of said chemicals.
The sleepy pen still has some use to it even with autoinjecors being brought back. They come with zombie powder, and can be hidden in a PDA, also they look like a pen and on examine dont show off you got the sneaky stinger in your pocket.

If this is really about making the sleepy pen not obsolete by a first aid kit, then buff the fucking sleepy pen.

Hardsuits are already super desirable after the recent viro rework, having near total immunity to chemical injections is too much.

**Comment by robosperg** [comment]
just for curiosity, can the emojiocracy vote on the idea of a medbay machine to fill autoinjectors with different chem payloads?

**Comment by Blithering** [comment]
@robosperg I like it, maybe have a set but upgradable list of chems like sleepers do.
@FudgePucker O2 kits also come with a syringe, making them a one stop shop. How would you even buff the tricky pen to be a worthwhile upgrade over chloral autoinjectors?

**Comment by Kurfursten** [comment]
It should be able to reset spent ones too

**Comment by FudgePucker** [comment]
Refillable sleepy pens.

**Comment by robosperg** [comment]
emojiocracy has spoken. 38% in favor isn't enough for me to bitch for a server vote.

**Pull request closed without merging**